crates/ruff_cli/src/diagnostics.rs

```diff
@@ -487,80 +508,3 @@ pub(crate) fn lint_stdin(
         notebook_indexes,
     })
 }
-
-#[derive(Debug)]
-pub(crate) struct LintSource(pub(crate) SourceKind);
```

Comment:
I removed this, and moved the logic into `SourceKind`, so that it can be used for formatting too.